superset-frontend/src/components/ErrorMessage/ErrorAlert.tsx

```diff
@@ -82,7 +82,7 @@ const LeftSideContent = styled.div`
 
 interface ErrorAlertProps {
   body: ReactNode;
-  copyText?: string;
+  copyText?: string | JSX.Element;
```

Comment:
is this change intentional?

Reply:
Type for `copyText` becomes invalid if `JSX.Element` type is not set. Following error gets thrown in the `ErrorMessageWithStackComponent`:

```
(JSX attribute) copyText?: string | undefined
Type 'Element' is not assignable to type 'string'.ts(2322)
ErrorMessageWithStackTrace.tsx(33, 3): The expected type comes from property 'copyText' which is declared here on type 'IntrinsicAttributes & Props' 
```

Reply:
Gotcha, thanks!